### ISSUE TITLE
Exact supported Browserversions

### ISSUE THREAD

**Issue by eichiflSICKAG** [angular/angular#54195]
### Describe the problem that you experienced

The docs state, that the 2 most recent chrome versions are supported. As chrome releases in a 4 week releasecycle it is pretty hard to determine which exactly are meant. 

### Enter the URL of the topic with the problem

https://angular.io/guide/browser-support

### Describe the actions that led you to experience the problem

Customers ask us which browsers they need to use. As we are in an industrial environment, it's not given, they are always on the latest browsers

### Describe what you want to experience that would fix the problem

It would make sense to state the exact minimum browser version alongside the releases angular version.

**Comment by json-derulo** [comment]
It would be an immense effort to determine the "minimum required version". Basically Angular would need to start with the current browser version and repeatedly downgrade Chrome and for each version run their tests to see if there are any errors. But even if there are no errors in the test scenarios, this doesn't say much because the application's code or third party dependencies could cause issues with older browser versions. IMHO it doesn't make any sense to mention some kind of "minmum required version".

If you need to ensure compatibility with older Chrome versions, you should add custom tests to your application which are to be executed on older Chrome versions.

**Comment by eichiflSICKAG** [comment]
yes right, for me it would be sufficient to just explicitly note the last two chrome versions when releasing
